webservices/resources/operations_log.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+from flask_apispec import doc
+from webservices import args
+from webservices import docs
+from webservices import utils
+from webservices import schemas
+from webservices.common import models
+from webservices.exceptions import ApiError
+from webservices.common.views import ApiResource
+
+logger = logging.getLogger(__name__)
+
+
+@doc(tags=['filings'], description=docs.OPERATIONS_LOG)
+class OperationsLogView(ApiResource):
+    model = models.OperationsLog
+    schema = schemas.OperationsLogSchema
+    page_schema = schemas.OperationsLogPageSchema
+
+    filter_multi_fields = [
+        ('candidate_committee_id', models.OperationsLog.candidate_committee_id),
+        ('beginning_image_number', models.OperationsLog.beginning_image_number),
+        ('report_type', models.OperationsLog.report_type),
+        ('report_year', models.OperationsLog.report_year),
+        ('form_type', models.OperationsLog.form_type),
+        ('amendment_indicator', models.OperationsLog.amendment_indicator),
+    ]
+
+    # filter_match_fields = [
+    #     ('amendment_indicator', models.OperationsLog.amendment_indicator),
+    # ]
+
+    @property
+    def args(self):
+        default_sort = ['-report_year']
+        return utils.extend(
+            args.paging,
+            args.operations_log,
+            args.make_multi_sort_args(
+                default=default_sort,
+            ),
+        )
+
+    def build_query(self, *args, **kwargs):
+        try:
+            query = super().build_query(*args, **kwargs)
+            # get the records whose status is verified
+            results = query.filter_by(status_num=1)
+        except:
+            raise ApiError("Unexpected Server Error", 500)
```

Comment:
This hides the root cause of the exception and makes it difficult to diagnose the underlying issue.

Reply:
are you suggesting to take off the entire try/except block from here, since we have a master exception handler in rest.py?

Reply:
Either that, or if we choose to catch the exception here, we should at least log what it was and not suppress it. When we used this pattern for the coverage dates endpoint, this handler made it difficult to debug.

Reply:
The master exception handler is handing the exceptions, and giving meaningful error message. I have removed the try/except block from this file.